src/components/error-message/index.md.njk

```diff
@@ -21,7 +21,9 @@ Use standard messages for different components.
 
 ## When not to use this component
 
-Do not use error messages to tell users that they are not eligible or do not have permission to do something. Instead, take them to a page which tells them they’re not eligible and gives them useful information about what to do next.
+Do not use error messages to tell a user that they are not eligible or do not have permission to do something. Or to tell them about a lack of capacity or other problem the user cannot fix - because the problem is with the service rather than with the information the user has provided.
```

Comment:
I wonder whether we could use bullets to avoid the sentence starting with 'Or' (which is in addition to the 'or' in the first sentence). Something like this?

Do not use error messages to tell a user:

- that they are not eligible
- that they do not have permission to do something
- about a problem with the service rather than the information they've provided - like a lack of capacity

Reply:
I think you lose some of the sense of `about a problem with the service rather than the information they've provided - like a lack of capacity` by including it in a bullet point.

Is it a problem to start a sentence with 'or'?

Reply:
Sure - just a suggestion.

I've used 'or' (and 'and') myself at the start of sentences in my own writing, but it felt a bit odd to me on GOV.UK. Might just be me!